### DISCUSSION TITLE
Gamepad still detected when turned off

### DISCUSSION THREAD

**Discussion by ghost** [MonoGame/MonoGame#8467] [Inputs]
Hello there,

When my wireless gamepad is turned off but the wireless adapter is plugged in, GamePad.GetState considers that the gamepad is connected and active.

Is there anything I can do to fix this ? I would like to immediately pause the game if the Gamepad runs out of battery for example.

Have a nice day.

**Comment by KelsamGames** [comment]
Depends on the gamepad. It is up to the device driver to report that it is disconnected. For gamepads that have a dedicated wireless dongle, it is unlikely that it is ever disconnected according to Windows (or whatever else) and therefore no way for you (MonoGame via SDL, DirectInput, etc) to detect it.

**Reply by ghost** [reply]
Hey thank you for your quick reply !

Arf this is exactly what I feared. Of course if the OS believes that the gamepad is connected there is nothing I can do about it :-/
